addons/libs/lists.lua

```diff
@@ -312,9 +312,11 @@ function list.clear(l)
 end
 
 function list.copy(l, deep)
-    deep = deep or true
+    if deep == nil then 
+        deep = true
+    end
```

Comment:
Good catch, this can be `deep = deep ~= false and true`. Can you please also adjust it in `tables.lua` and `sets.lua`?